phpstan.neon.dist

```diff
@@ -2,7 +2,8 @@ includes:
 	- vendor/phpstan/phpstan-phpunit/extension.neon
 	- vendor/phpstan/phpstan-phpunit/rules.neon
 parameters:
-    level: 1
+    level: 5
+    reportUnmatchedIgnoredErrors: false
```

Comment:
Which use case requires to have ```reportUnmatchedIgnoredErrors``` ?

Reply:
I have set that reportUnmatchedIgnoredErrors: false because travis build failed for symfony: 5.0 and for now, I have no other solution. Check failed build:
https://travis-ci.org/github/eMAGTechLabs/RabbitMqBundle/builds/694626432

and see logs:
https://travis-ci.org/github/eMAGTechLabs/RabbitMqBundle/jobs/694626436

Reply:
OK I see. This is probably because of this weird stuff : https://github.com/symfony/symfony/blob/4.3/src/Symfony/Contracts/EventDispatcher/EventDispatcherInterface.php

I don't think there is a solution, or we should check if we can use the _Symfony\Contracts\EventDispatcher\EventDispatcherInterface_ instead.

Reply:
Yes, It is because of that comment in dispatch function params: https://github.com/symfony/symfony/blob/4.3/src/Symfony/Contracts/EventDispatcher/EventDispatcherInterface.php 
And the difference in https://github.com/symfony/symfony/blob/5.0/src/Symfony/Contracts/EventDispatcher/EventDispatcherInterface.php.
So, without that reportUnmatchedIgnoredErrors: false, travis build will fail for Symfony 5.0. 
Unfortunately, I have no other solution for now. 
Check travis builds https://travis-ci.org/github/eMAGTechLabs/RabbitMqBundle/builds/694626432